vehicles/src/modules/special-auth/dto/request/create-special-auth.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+
+import { IsOptional } from 'class-validator';
+import { NoFeeType } from 'src/common/enum/no-fee-type.enum';
+
+export class CreateSpecialAuthDto {
```

Comment:
Is this file used anywhere? Seems to have no connection with any controller?

Reply:
I'm not too sure, you might want to ask Garima as she worked on that branch, and I'll simply merged her branch into mine

Reply:
I had removed this file. please merge main into your branch

Reply:
Removed this file